stdlib/asm/sys.masm

```diff
@@ -0,0 +1,26 @@
+proc.stack_cleanup_proc.16
+    popw.local.0
+    popw.local.1
+    popw.local.2
+    popw.local.3
+    push.env.sdepth
+    push.16
+    u32gt.unsafe
+    while.true
+        dropw
+        push.env.sdepth
+        push.16
+        u32gt.unsafe
+    end
+    loadw.local.3
+    swapw.3
+    loadw.local.2
+    swapw.2
+    loadw.local.1
+    swapw.1
+    loadw.local.0
+end
+
+export.stack_cleanup
+    exec.stack_cleanup_proc
```

Comment:
We should be able to put the actual code in here - no need to have two separate procedures.

Reply:
Thank you. I thought I tried this earlier unsuccessfully but must have been mistaken.